File: Basics_with_Python/PythonBasicCourse/3_Conditional Statements Advanced/8_Conditional_Statements_Advanced - Exercise - Lab/more_exercises/04. Car To Go.py
budged = float(input())
season = input()
car_class = 0
car = 0
car_price = 0

if budged <= 100:
    car_class = "Economy class"
    if season == "Summer":
        car = "Cabrio"
        car_price = budged * 0.35
    elif season == "Winter":
        car = "Jeep"
        car_price = budged * 0.65

elif 100 < budged <= 500:
    car_class = "Compact class"
    if season == "Summer":
        car = "Cabrio"
        car_price = budged * 0.45
    elif season == "Winter":
        car = "Jeep"
        car_price = budged * 0.80

elif budged > 500:
    car_class = "Luxury class"
    car = "Jeep"
    car_price = budged * 0.9

print(car_class)
print(f"{car} - {car_price:.2f}")

File: Basics_with_JavaScript/78_Conditional_Statements_Advanced/More_Exersises/04._Car_To_Go.js
function vladi(input){

    let budged = Number(input[0])
    let season = (input[1])
    let classCar = ""
    let typeCar = ""
    let priceCarPercentFromBudged = 0
    let priceCar = 0
    
    if (budged<= 100) {
        classCar = "Economy class";
        switch (season) {
            case "Summer":
                typeCar = "Cabrio";
                priceCarPercentFromBudged = 35;
                break;
            case "Winter":
                typeCar = "Jeep";
                priceCarPercentFromBudged = 65;
                break;
        }
    } else if (budged > 100 && budged <= 500) {
        classCar = `Compact class`;
                if (season === "Summer") {
                    typeCar = "Cabrio";
                    priceCarPercentFromBudged = 45;
                } else if (season === "Winter") {
                    typeCar = "Jeep";
                    priceCarPercentFromBudged = 80;
                }
    } else if (budged > 500) {
        classCar = "Luxury class";
        typeCar = "Jeep";
        priceCarPercentFromBudged = 90;
    }

    priceCar = (budged * priceCarPercentFromBudged / 100).toFixed(2);

    console.log(classCar);
    console.log(`${typeCar} - ${priceCar}`);

    }
   
    vladi(["450","Summer"]);
    vladi(["450","Winter"]);
    vladi(["1010","Summer"]);
    vladi(["99.99","Summer"]);
    vladi(["1010","Winter"]);
    vladi(["70.50","Winter"]);